util/linux_install.sh

```diff
@@ -95,6 +95,18 @@ elif grep ID /etc/os-release | grep -q gentoo; then
 		echo "Quitting..."
 	fi
 
+elif grep ID /etc/os-release | grep -q sabayon; then
+	sudo equo install \
+		app-arch/unzip \
+		app-arch/zip \
+		app-mobilephone/dfu-util \
+		net-misc/wget \
+		sys-devel/gcc \
+		sys-devel/crossdev dev-embedded/avrdude
+	sudo crossdev -s4 --stable --g =4.9.4 --portage --verbose --target avr
+	echo Done!
```

Comment:
```suggestion
	echo 'Done!'
```
`!` has a special meaning in certain shells and certain modes. Even though it shouldn't cause problems with this particular string, I'd put it in hard quotes just in case. (Also do this on line 93.)

Reply:
Should I amend the previous commit? I ask because I don't want to commit-spam.

Reply:
Either way is fine. In this case I'd go with a new commit (e.g. `Put echo string in quotes`).

Reply:
@BlitzKraft Commit spam is 100% acceptable. 

If there is too much, we generally squash the commit to clean it up.  But 2-3 commits is a small amount, and may not need to be squashed. 

So, feel free to add a bunch more commits, if needed.